### ISSUE TITLE
Add Additional information to Agencies API Response

### ISSUE THREAD

**Issue by davidpickering** [midohiofoodbank/freshtrak-pantry-finder-api#15]
Story - Agency Operator: I run an agency, and when people view my information on FreshTrak.com, there is some information I want them to know. This information can include details about parking, how to find our location, what to bring, what to expect, etc...

Example: [Worthington Resource Pantry](https://www.worthingtonresourcepantry.org/get-help)

Data:
- The note we want to display is in the events.pub_desc_long field
- The data is only populated for four events, both should be results of an API call to 43123.

1. Event ID == 23	20896	Grove City Food Pantry
2. Event ID ==  5563	0	Drive Thru
3. Event ID ==  5574	0	Home Delivery
4. Event ID ==  5596	0	Drive Thru Wednesday Evening

- The data should be passed as a property of the events
- The data will only be shown to a customer if NOT NULL

Example 1: 

![image](https://user-images.githubusercontent.com/11682225/81363405-40dc1e00-90b1-11ea-80e3-64c9f203872f.png)

Example 2: 

![image](https://user-images.githubusercontent.com/11682225/81363429-4b96b300-90b1-11ea-9bd2-5d4f317118fa.png)

**Comment by PhilNorman2** [comment]
We have a convention of returning an empty string rather than a null for string data elements in the API response.  To do this we need to rename the JSON output attribute, preferably in plain English.  Below is an example with the field renamed public_description.  @davidpickering, do you have a preferred name for this output field?  I didn't see a pub_desc_short field in the events table.

<pre>
"events": [
                {
                    "id": 686,
                    "address": "5075 CLEVELAND AVE  ",
                    "city": "COLUMBUS",
                    "state": "OH",
                    "zip": "43231",
                    "agency_id": 495,
                    "latitude": "40.07427271",
                    "longitude": "-82.95193391",
                    "name": "Pantry",
                    "service": "Choice Pantry",
                    "estimated_distance": 15.13,
                    "public_description": "",
</pre>

**Comment by davidpickering** [comment]
I don't have a big preference. In the spirit of generalization and not having to explain it, this data will be fed into the UI through the "more details" button, and also shown on the "appointment confirmation" page. So, I think we could generally call it "event_details" and that would be a little more explanatory, and true to the UI. (Although if it ends up as is I am fine too)

**Comment by PhilNorman2** [comment]
event_details it is:

<pre>
"events": [
                {
                    "id": 5563,
                    "address": "3960 BROOKHAM DR  ",
                    "city": "GROVE CITY",
                    "state": "OH",
                    "zip": "43123",
                    "agency_id": 6,
                    "latitude": "39.8862352",
                    "longitude": "-83.05538713",
                    "name": "Drive Thru",
                    "service": "Prepack Pantry",
                    "estimated_distance": 1.39,
                    "event_details": "This is a drive thru distribution. Please, no more than two families being served per vehicle. Please pre-register on FreshTrak.com. ",
</pre>

**Comment by PhilNorman2** [comment]
To test this I had to update the schema.  So will probably do a separate PR with the schema and anything impacted by it (already broke my Foodbank tests).   After that I was thinking of combining this issue with issue #16 in a single PR.

**Comment by PhilNorman2** [comment]
Test results for query: http://localhost:8888/api/agencies?zip_code=43123

Event ID == 23 20896 Grove City Food Pantry
Passed:
"event_details": "This is our normal pantry. You must get an appointment from HandsOn "

Event ID == 5563 0 Drive Thru
Passed:
"event_details": "This is a drive thru distribution. Please, no more than two families being served per vehicle. Please pre-register on FreshTrak.com. "

Event ID == 5574 0 Home Delivery
Failed:  This event did not appear in the response.  
Cause: event_id 5574 is not present in event_service_geographies.

Event ID == 5596 0 Drive Thru Wednesday Evening
Passed:
"event_details": "This is a drive thru distribution. Wear a mask and make sure your trunk is cleaned out. Please, no more than two families being served per vehicle. Please pre-register on FreshTrak.com."

**Note:** The following event_details message appears for Event ID == 438 accessible from the query: http://localhost:8888/api/agencies?zip_code=43046:
"event_details": "ABC dfadkifja   covid bad!"
